std/mem.zig

```diff
@@ -0,0 +1,21 @@
+import "syscall.zig";
+import "std.zig";
+
+pub fn malloc(bytes: isize) -> ?&u8 {
+    if (bytes > 4096) {
```

Comment:
I wonder what this number depends on. It would be interesting to read the source code of `getpagesize`.